app/src/main/java/org/oppia/android/app/testing/DrawableBindingAdaptersTestActivity.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.android.app.testing
+
+import android.os.Bundle
+import org.oppia.android.R
+import org.oppia.android.app.activity.ActivityComponentImpl
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+
+/** Test activity for DrawableBindingAdapters. */
```

Comment:
```suggestion
/** Test activity for [DrawableBindingAdapters]. */
```

Reply:
This way we can directly jump to that particular file. 
As `DrawableBindingAdapters` is a java file, if it doesn't work, ignore it, else put these square bracket. 

Take a look at example - https://github.com/oppia/oppia-android/blob/07bee869e2aecb6fbb704d5ad9af0c885800f4d7/app/src/main/java/org/oppia/android/app/testing/HomeTestActivity.kt#L8

Reply:
@anandwana001 Actually, I checked for other data binding test activities but none of them are doing this, so I just left as it is. It can be confirmed if you look into these files. I have linked these 2, but there are more.
https://github.com/oppia/oppia-android/blob/3a0afb4eb8222d4ff3acd2e2e660b85911d35229/app/src/main/java/org/oppia/android/app/testing/StateAssemblerMarginBindingAdaptersTestActivity.kt#L16-L15

https://github.com/oppia/oppia-android/blob/3a0afb4eb8222d4ff3acd2e2e660b85911d35229/app/src/main/java/org/oppia/android/app/testing/ViewBindingAdaptersTestActivity.kt#L16-L15

@BenHenning Please confirm what to do.

Reply:
This reference won't work since the class isn't being imported. I suggest either using backticks, i.e.:
```
``DrawableBindingAdapters``
```

Or, fully qualify the type: ``[org.oppia.android.app.databinding.DrawableBindingAdapters]``

(Note that you can check if a Markdown reference is correct by control clicking on the class name; if it doesn't open or has a squiggly underneath it, the reference isn't correct).